### DISCUSSION TITLE
Adding german translation

### DISCUSSION THREAD

**Discussion by peterbeck** [convos-chat/convos#767] [Q&A] [answered]
Hi there,

to add a german translation, can I just copy assets/l18n/en.po, do translate msgstr, rename it to ge.po (or whatever) and make a pull request ?

Thanks
Peter

**Comment by jhthorsen** [comment, marked as answer]
Hi,

Yes, that is correct. I think the country code for Germany is "de" though, so please call it "assets/l18n/de.po". Also if you start the server locally i development mode, it will read the translations file each time you hit "refresh" in your browser.

https://convos.chat/doc/develop has instructions on how to set up a local environment.
